packages/server-client/src/client.ts

```
@@ -376,79 +376,86 @@ export class IfcServerClient {
-            case 'complete':
-              stats = event.stats;
-              metadata = event.metadata;
-              symbolic_data = event.symbolic_data;
-              const totalTime = performance.now() - uploadStart;
-              console.log(`[client] Stream complete: ${total_meshes} meshes in ${totalTime.toFixed(0)}ms`);
-              break;
+              case 'complete':
+                stats = event.stats;
+                metadata = event.metadata;
+                symbolic_data = event.symbolic_data;
+                const totalTime = performance.now() - uploadStart;
```

Biome (2.5.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)